lib/src/layout_element.dart

```diff
@@ -82,19 +141,9 @@ class TableSectionLayoutElement extends LayoutElement {
 
   @override
   Widget toWidget(RenderContext context) {
+    // Not rendered; TableLayoutElement will instead consume its children
```

Comment:
If this won't be consumed, is there any way that we can get rid of it?

Reply:
Yes, several alternatives. For example, we could make `TableSectionLayoutElement` (and `TableRowLayoutElement`) not be their own types but simply `StyledElement`s and check their tag names where we use them (inside `TableLayoutElement`). I just tried to reduce the change surface as far as I could (but still added this comment as why the heck we do this).